transform/mattermost-analytics/models/staging/mm_calls_test_go/_mm_calls_test_go__sources.yml

```diff
@@ -0,0 +1,36 @@
+version: 2
+
+sources:
+  - name: mm_calls_test_go
+    database: 'RAW'
+    schema: mm_calls_test_go
+    loader: Rudderstack
+    description: |
+      Mattermost Calls telemetry data. Stored using Rudderstack.
+      
+    tags: [ 'rudderstack' ]
+
+    # Omitting tables with 20 or less rows.
+    tables:
+      - name: call_notify_admin
```

Comment:
Missing `identifies` and `users` table is a bit suspicious. Not much we can do but worth mentioning.

Reply:
Yes, this was weird, maybe calls work differently? we can check with Claudio later